src/network/networkd-radv.c

```diff
@@ -782,6 +864,34 @@ void network_drop_invalid_route_prefixes(Network *network) {
                         route_prefix_free(p);
 }
 
+static int pref64_prefix_section_verify(pref64Prefix *p) {
+        if (section_is_invalid(p->section))
+                return -EINVAL;
+
+        if (!IN_SET(p->prefixlen, 32, 40, 48, 56, 64, 96))
+                return log_warning_errno(SYNTHETIC_ERRNO(EINVAL),
+                                         "%s: Invalid prefix length %u is specified in [IPv6PREF64Prefix] section. "
+                                         "Valid lengths are 32, 40, 48, 56, 64 and 96. Ignoring [IPv6PREF64Prefix] section from line %u.",
+                                         p->section->filename, p->prefixlen, p->section->line);
+
+        if (p->lifetime == 0)
```

Comment:
We should accept this. Please see @mweinelt's comment.

Reply:
yes dropped